src/OrbitVulkanLayer/SubmissionTracker.h

```diff
@@ -265,6 +279,16 @@ class SubmissionTracker : public VulkanLayerProducer::CaptureStatusListener {
     CHECK(text != nullptr);
     bool marker_depth_exceeds_maximum;
     {
+      if (!command_buffer_to_state_.contains(command_buffer)) {
+        if (!logged_marker_begin_when_initial_) {
+          ERROR(
+              "Calling vkCmdDebugMarkerBeginEXT/vkCmdBeginDebugUtilsLabelEXT on a command buffer "
+              "that is in the initial state (i.e. either freshly allocated or reset with "
+              "vkResetCommandBuffer).");
+          logged_marker_begin_when_initial_ = true;
+        }
+        return;
```

Comment:
I guess this return here (and below, and the "continue") is the actual fix for the crash?